Description: Configures build (using gcj) to behave more like sun-jdk-5 
 in terms of error messages.
Author: james.page@canonical.com
Forwarded: not-required
Index: c3p0/build.xml
===================================================================
--- c3p0.orig/build.xml	2011-02-25 11:52:44.775376257 +0000
+++ c3p0/build.xml	2011-02-25 12:02:54.804996643 +0000
@@ -208,7 +208,7 @@
 	     source="${c3p0.target.version}"
 	     target="${c3p0.target.version}"
              classpathref="codegen-classpath"
-             debug="true">
+             debug="true" nowarn="on" deprecation="off">
 			<src>
 				<pathelement location="${build.codegen.dir}" />
 				<pathelement location="${java.src.dir}" />
@@ -253,7 +253,7 @@
 	     source="${c3p0.target.version}"
 	     target="${c3p0.target.version}"
              classpathref="build-classpath" 
-             debug="on">
+             debug="on" nowarn="on" deprecation="off">
 			<sourcepath>
 				<pathelement location="${build.codegen.dir}" />
 				<pathelement location="${java.src.dir}" />
@@ -275,7 +275,7 @@
 	     source="${c3p0.target.version}"
 	     target="${c3p0.target.version}"
              classpathref="build-classpath" 
-             debug="on">
+             debug="on" nowarn="on" deprecation="off">
 			<sourcepath>
 				<pathelement location="${build.codegen.dir}" />
 			</sourcepath>
@@ -290,7 +290,7 @@
 		<javac destdir="${build.classes.dir}" 
 	     source="${c3p0.target.version}"
 	     target="${c3p0.target.version}"
-             debug="on">
+             debug="on" nowarn="on" deprecation="off">
 			<classpath>
 				<path refid="build-classpath" />
 				<pathelement location="${log4j.jar.file}" />
@@ -311,7 +311,7 @@
 		<javac destdir="${build.classes.dir}" 
 	     source="${c3p0.target.version}"
 	     target="${c3p0.target.version}"
-             debug="on">
+             debug="on"  nowarn="on" deprecation="off">
 			<classpath>
 				<path refid="build-classpath" />
 				<pathelement path="${junit.jar.file}" />
Index: c3p0/relproj/debuggen/build.xml
===================================================================
--- c3p0.orig/relproj/debuggen/build.xml	2011-02-25 12:04:02.992661915 +0000
+++ c3p0/relproj/debuggen/build.xml	2011-02-25 12:04:30.047767796 +0000
@@ -32,7 +32,7 @@
       <mkdir dir="${build.classes.dir}" />
       <javac srcdir="${java.src.dir}" 
              destdir="${build.classes.dir}" 
-             debug="on" />
+             debug="on"  nowarn="on" deprecation="off"/>
    </target>
 
    <target name="jar" depends="compile">
